docs/source/resources.rst

```diff
@@ -48,6 +48,34 @@ When we submit tasks to the cluster we specify constraints per task
    final = client.submit(aggregate, processed, resources={'MEMORY': 70e9})
 
 
+Resources are per worker process
+--------------------------------
+
+If you are using ``dask-worker --nprocs <nprocs>`` the resource will be applied
+to each of the ``nprocs`` workers. Suppose you have 2 GPUs on your machine, if
+you want to use two worker processes, you have 1 GPU per worker process so you
+need to do something like this::
+
+   dask-worker scheduler:8786 --nprocs 2 --resources "GPU=1"
+
+Here is an example that illustrates how to use resources to ensure each task is
+run inside a separate process, which is useful to execute non thread-safe tasks
+or tasks that uses manages multithreading internally::
+
+   dask-worker scheduler:8786 --nprocs 3 --nthreads 2 --resources "process=1"
+
+With the code below, there will be at most 3 tasks running concurrently and
+each task will run in a separate process::
+
+.. code-block:: python
+
+   from distributed import Client
+   client = Client('scheduler:8786')
+
+   futures = [client.submit(non_thread_safe_function, arg,
+                            resources={'process': 1}) for arg in args]
```

Comment:
I think that these lines may need one more space indenting them.  By looking at the rendered view on GitHub you should be able to see that they're a bit odd at the moment.

Reply:
Done. On a good day, I am able to type a `tar` command on the first attempt (re xkcd) but that's definitely not the case for reST ...

Finding mistakes via github is a great tip by the way, I never thought of that.